src/outfit.ts

```diff
@@ -352,6 +352,9 @@ export class Outfit {
    * @returns True if one of the things is equipped, and false otherwise.
    */
   public equipFirst(things: Item[] | Familiar[], slot?: Slot): boolean {
+    // some() returns false on an empty array, yet every() returns true.
+    // This keeps behavior consistent between slotful and slotless equipping.
+    if (things.length < 1) return true;
```

Comment:
I would use .length === 0 instead, probably

I also don't think his is necessarily inconsistent behavior, but it _is_ behavior that I don't like